### PR TITLE
Support pileup

### PR DESCRIPTION
**Pull request by zaborowska**
- adding event merging+mixing to create pileup events (can be also extended in send.py so that signal event is additionally added)
- adding tool to estimate the pileup noise in ecal clusters
- print warnings if wrong config is used (some options may require code newer than the release)
- use path to the detector from the actual local installation if it is specified
- adding new calib constants for the future generation of sim

### CODE REVIEW

File: config/recTopoClusters.py [hunk 1]
```diff
@@ -17,17 +20,21 @@
 print "=================================="
 num_events = simargs.numEvents
 input_name = simargs.inName
-output_name = 'clusters.root'
```
Comment (CoralieNeubueser) [inline config/recTopoClusters.py]: Ok, here you change the output file name..

File: python/send.py [hunk 2]
```diff
@@ -438,18 +524,20 @@ def getJobInfo(argv):
...
-        elif '--recTopoClusters' in sys.argv:
-            frun.write('python %s/python/Convert.py $JOBDIR/clusters.root $JOBDIR/%s\n'%(current_dir,outfile))
-            frun.write('python /afs/cern.ch/work/h/helsens/public/FCCutils/eoscopy.py $JOBDIR/%s %s\n'%(outfile,outdir))
-            reco_path = outdir.replace('/ntup/', '/reco/')
-            if not ut.dir_exist(reco_path):
-                os.system("mkdir -p %s"%(reco_path))
-            frun.write('python /afs/cern.ch/work/h/helsens/public/FCCutils/eoscopy.py $JOBDIR/clusters.root %s\n'%(reco_path+'/'+outfile))
-            
+        elif '--recTopoClusters' in sys.argv or '--recSlidingWindow' in sys.argv:
+            frun.write('python %s/python/Convert.py $JOBDIR/%s $JOBDIR/clusters.root\n'%(current_dir,outfile))
```
Comment (CoralieNeubueser) [inline python/send.py]: so, here you need to change "clusters.root" to the same output file name.

**Comment by zaborowska** [comment]
Hi @CoralieNeubueser , I did update paths:
https://github.com/FCC-hh-framework/FCCSimJobs/pull/28/commits/53ede7f18f2a50898021a38453edae2e3e24b80f
I think we should always stick to same naming pattern, renaming it for the intermidiate files only.

**Comment by zaborowska** [comment]
clusters.root is now the output path of your script so it is ok, it gots copied later on to /ntup/

**Comment by CoralieNeubueser** [comment]
Ah, ok, now I see, you switched the names for reco and ntup files..

File: config/recSlidingWindow.py [hunk 3]
```diff
@@ -4,8 +4,12 @@
 simparser.add_argument('--inName', type=str, help='Name of the input file', required=True)
 simparser.add_argument('--outName', type=str, help='Name of the output file', required=True)
 simparser.add_argument('-N','--numEvents', type=int, help='Number of simulation events to run', required=True)
+simparser.add_argument('--detectorPath', type=str, help='Path to detectors', default = "/cvmfs/fcc.cern.ch/sw/releases/0.9.1/x86_64-slc6-gcc62-opt/linux-scientificcernslc6-x86_64/gcc-6.2.0/fccsw-0.9.1-c5dqdyv4gt5smfxxwoluqj2pjrdqvjuj")
```
[the remark this answers is not in the document]
Comment (zaborowska) [reply config/recSlidingWindow.py]: done

**Pull request merged**